### ISSUE TITLE
Problems with upgrading from 1.5.2 to 1.8.0

### ISSUE THREAD

**Issue by jiakangning** [single-spa/single-spa-vue#36]
Upgrade version to be found today：
 **about v1.8.0 is:** 

`DOMException: Failed to execute 'querySelector' on 'Document': [Object HTMLDivElement] .single-spa-container` is not a valid selector`

**Found after debugger the source code， the user of this method is an array like object rather than a standard element object**

These are my findings. I hope to give you some suggestions

**Comment by joeldenning** [comment]
Looks like you might be using jquery to select the div? I didn't know that Vue supported passing in jquery array wrappers for elements. I'll put together a fix.

**Comment by joeldenning** [comment]
It looks like my guess about jquery was incorrect. [This code sandbox](https://codesandbox.io/s/nostalgic-cherry-6dbrw) shows that vue itself isn't okay with a jquery wrapper.

Re-reading your original issue, "array-like" probably refers to [NodeList](https://developer.mozilla.org/en-US/docs/Web/API/NodeList). I'll check if that one works with Vue next

**Comment by joeldenning** [comment]
Looks like NodeLists also don't work with vue@2.6.11. See [this code sandbox](https://codesandbox.io/s/green-leaf-d24i6) that shows that.

Can you provide more detail? What exactly are you passing in as `appOptions.el`? Can you share the code that calls `singleSpaVue({...})`

**Comment by iamfrontender** [comment]
Hi, I've stumbled around this exact error today. The steps to reproduce were to use `singleSpaValue(...)` as a Parcel in a following fashion:
```javascript
singleSpa.mountRootParcel(parcelConfig, {
  domElement: document.querySelector('main'),
})
```
That results that `appOptions.el` is an actual HTMLElement which being overwritten as a string at: 
https://github.com/single-spa/single-spa-vue/commit/b039bf8f6123b3aa489cf085bc793bbd7458e103#diff-0d2771b67ca1ace18fd46aee0897026eR88

I've left an additional comment in the commit itself relying the same information, as that's what I've found on Google researching this error.

Thanks for all your efforts on maintaining the package!
